README_GENERATE.md

```diff
@@ -23,8 +23,22 @@ Then:
 
 ## Generate the library
 
+There is a bug with version 4.3.1 which make the following request generate a broken API package:
+
     openapi-generator generate -g python -i swagger.json --package-name pds.api_client --additional-properties=packageVersion=0.2.0
-     
+
+A fork and pull request has been made to correct that. To use the patched version, do:
+
+    git clone https://github.com/tloubrieu-jpl/openapi-generator.git
+    cd  openapi-generator
+    ./mvnw clean package
```

Comment:
Should this be `./mvn clean package`?

Reply:
Yes that could be but they have there own maven wrapper which I used in my test.

Reply:
I mean "they" the original authors of the openapi-generator repository.